src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -383,18 +383,35 @@ export default class AircraftController {
     }
 
     /**
+     * Finds an aircraft by its callsign
+     *
      * @method findAircraftByCallsign
      * @param  {string} [callsign='']
      * @return {AircraftModel|null}
      */
     findAircraftByCallsign(callsign = '') {
-        if (callsign === '') {
+        if (!callsign) {
             return null;
         }
 
         return _find(this.aircraft.list, (aircraft) => aircraft.callsign.toLowerCase() === callsign.toLowerCase());
     }
 
+    /**
+     * Finds an aircraft by its internal id
+     *
```

Comment:
_why_ would be need to find and aircraft by an internal ID?  don't we reference aircraft by their callsigns? 

I know the answer here, but we should document this reason for our future selves.

Reply:
That's good because I have no idea why we use the internal ID in the runway queue. 🙈

Reply:
I don't remember either.. I think it was some circular dependency issue, where we added the id because we couldn't store the AircraftModel? I dunno.